HelpSource/Classes/SortedList.schelp

```diff
@@ -5,7 +5,7 @@ categories::Collections>Ordered
 CLASSMETHODS::
 
 method::new
-Creates a SortedList with the initial capacity given by strong::size:: and a comparison strong::function::.
+Creates a SortedList with the initial capacity given by strong::size:: and a comparison strong::function::. The SortedList can contain equal elements more than once.
```

Comment:
If we really want to state that SortedList can contain duplicates (maybe superfluous IMO since this is what most Collections do), then we should say "equal or identical"